dns-discovery/build.gradle

```diff
@@ -8,9 +8,3 @@ dependencies {
     compile 'dnsjava:dnsjava:2.1.7'
     testCompile project(':integration-testing')
 }
-
-sourceSets {
```

Comment:
afaik there was a problem once because the resources directory was getting created even though there are no resources, this is the reason why we do this. did that change?